tools/syz-declextract/run.go

```diff
@@ -72,15 +112,61 @@ func worker(outputs chan output, files chan string, binary, compilationDatabase
 
 		cmd := exec.Command(binary, "-p", compilationDatabase, file)
 		stdout, err := cmd.Output()
-		var stderr []byte
+		var stderr string
 		if err != nil {
 			var error *exec.ExitError
 			if errors.As(err, &error) {
-				stderr = error.Stderr
+				stderr = string(error.Stderr)
 			} else {
-				stderr = []byte(err.Error())
+				stderr = err.Error()
+			}
+		}
+		outputs <- output{string(stdout), stderr}
+	}
+}
+
+func renameSyscall(kernelDir string) func(string) string {
+	var rename = make(map[string]string)
+	var isValidSyscall = make(map[string]bool)
+	for _, arch := range targets.List[targets.Linux] {
+		filepath.Walk(kernelDir+arch.KernelHeaderArch, func(path string, info fs.FileInfo, err error) error {
+			if strings.HasSuffix(path, ".tbl") {
```

Comment:
Reduce nesting level by returning early (the same way you for links already, so also will be more consistent):
```
if !strings.HasSuffix(path, ".tbl") {
    return nil
}
```

Reply:
Done!